Fix QA Notice: Function `dbiFreeIndexSet' implicitly converted to pointer at transaction.c:2086
that will almost certainly crash on 64bit architectures.
Patch provided by Jeff Johnson <n3npq.jbj@gmail.com> on Rpm-devel list.
--- rpm-4.4.7/lib/transaction.c.orig	2007-01-20 15:44:30.275657635 +0100
+++ rpm-4.4.7/lib/transaction.c	2007-01-20 15:45:09.818625107 +0100
@@ -2083,7 +2083,7 @@
 		shared->isRemoved = (knownBad == ro);
 		shared++;
 	    }
-	    matches[i] = dbiFreeIndexSet(matches[i]);
+	    (void) dbiFreeIndexSet(matches[i]);
 	}
 	numShared = shared - sharedList;
 	shared->otherPkg = -1;
